packages/url-loader/src/plugins/named-transformations.ts

```diff
@@ -1,18 +1,50 @@
+import { z } from 'zod';
+
 import { PluginSettings } from '../types/plugins';
 
-export const props = ['transformations'];
+const NamedTransformationSchema = z.string();
+type NamedTransformation = z.infer<typeof NamedTransformationSchema>;
+
+export const pluginProps = {
+  namedTransformations: z.union([
+      NamedTransformationSchema,
+      z.array(NamedTransformationSchema)
+    ])
+    .describe(JSON.stringify({
+      text: 'Named transformations to apply to asset.',
+      url: 'https://cloudinary.com/documentation/image_transformations#named_transformations'
+    }))
+    .optional(),
+  // @todo: deprecate in favor of namedTransformations
```

Comment:
question: Shouldn't this be removed as it is a new version?

Reply:
it never got a deprecation notice in a previous version, that was my only concern, do you think its okay for me to just move forward with it given its a breaking release?

Reply:
Well, the safer option would be to make it deprecated first (but not with a todo but rather a deprecated notice) and in the next version remove this functionality (to give users the time to migrate):

```ts
/**
 * @deprecated should use instance of {@link `element`} instead or eventually (not recommended) {@link other-element}
 */
```